Cargo.toml

```diff
@@ -5,4 +5,5 @@ members = [
     "light-node",
     "rpc",
     "tendermint",
+    "tendermint-testgen"
```

Comment:
How about naming the folder just `testgen` for consistency with the other crates in the workspace? We would still keep `tendermint-testgen` as the crate name in `testgen/Cargo.toml`.

Reply:
yes, nice idea, applied:)